test/conformance/queue/urQueueFinish.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+#include <uur/fixtures.h>
+
+using urQueueFinishTest = uur::urQueueTest;
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urQueueFinishTest);
+
+TEST_P(urQueueFinishTest, Success) {
+    constexpr size_t buffer_size = 1024;
+    ur_mem_handle_t buffer = nullptr;
+    ASSERT_SUCCESS(urMemBufferCreate(context, UR_MEM_FLAG_READ_WRITE,
+                                     buffer_size, nullptr, &buffer));
+
+    std::vector<uint8_t> data(buffer_size, 42);
+    ASSERT_SUCCESS(urEnqueueMemBufferWrite(queue, buffer, /* blocking */ false,
+                                           0, 1024, data.data(), 0, nullptr,
+                                           nullptr));
+
+    ASSERT_SUCCESS(urQueueFinish(queue));
+}
+
+TEST_P(urQueueFinishTest, InvalidNullHandleQueue) {
+    ASSERT_EQ_RESULT(UR_RESULT_ERROR_INVALID_NULL_HANDLE,
```

Comment:
Is UR_RESULT_ERROR_INVALID_QUEUE something that could be tested as well?

Reply:
This is just the tests that we are porting from UnitUR. Some of them aren't as complete as we would like. I have #223 to track it.